IPCloneDetectorAndASNAnalyzer/IPCloneDetectorAndASNAnalyzer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+  @title IP Clones and ASN Analysis
+  @author Madrix
+  @description Displays some statistics about ASNs, IPs, and clones.
+  @contact 
+  @version 1.0
```

Comment:
You have a version mismatch with the version in the class name - these should match :D